src/plugins/discover/public/application/main/components/no_results/no_results_suggestions/no_results_suggestions.tsx

```diff
@@ -132,28 +140,35 @@ export const NoResultsSuggestions: React.FC<NoResultsSuggestionProps> = ({
       }
       body={body}
       actions={
-        <div
-          css={css`
-            min-block-size: ${euiTheme.size.xxl};
-          `}
-        >
-          {typeof occurrencesRange === 'undefined' ? (
-            !isTextBasedQuery(query) && <EuiLoadingSpinner />
-          ) : canExtendTimeRange ? (
-            <EuiButton
-              color="primary"
-              fill
-              onClick={extendTimeRange}
-              isLoading={isExtending}
-              data-test-subj="discoverNoResultsViewAllMatches"
-            >
-              <FormattedMessage
-                id="discover.noResults.suggestion.viewAllMatchesButtonText"
-                defaultMessage="View all matches"
-              />
-            </EuiButton>
-          ) : null}
-        </div>
+        !isTextBasedQuery(query) && isTimeBased ? (
+          <div
+            css={css`
+              min-block-size: ${euiTheme.size.xxl};
+            `}
+          >
+            {timeRangeExtendingStatus === TimeRangeExtendingStatus.notFound ? (
+              <EuiText color="subdued" data-test-subj="discoverSearchAllMatchesGivesNoResults">
+                <FormattedMessage
+                  id="discover.noResults.suggestion.searchAllMatchesButtonGivesNoResultsText"
+                  defaultMessage="Expanding the time range gives no results too."
```

Comment:
```suggestion
                  defaultMessage="No results were found."
```
I wonder if something simpler would suffice? Not sure if my suggestion is great either, wdyt @amyjtechwriter? This would be shown in the case that a search across the time range returns no results.

Reply:
@davismcphee What if instead we strikethrough the bullet text and show (No results) on the right like so:

<img width="1377" alt="image" src="https://github.com/elastic/kibana/assets/4016496/a6206340-5f68-4b2c-b01f-933b0479a8bb">

Reply:
Works for me! And as an added bonus it should prevent the layout shifting currently happening when the button is swapped out. We'll see what @jughosta thinks when she's back from PTO 🙂

Reply:
@davismcphee @andreadelrio I updated the code to include this suggestion. The layout shift unfortunately is still there:

![May-07-2024 12-22-43](https://github.com/elastic/kibana/assets/1415710/a6d76cc4-6f52-47e8-abe8-4a657d3a5b00)

For cases like a request timeout or shards failure, I added a toast but keeping the button. Wdyt? If we can, I guess it's better to explain what happened than only show "No results". 

![May-07-2024 13-27-12](https://github.com/elastic/kibana/assets/1415710/7571b17d-1c73-439f-b9f0-34aede8842e2)

Reply:
@jughosta you should add a min-height to the container to avoid the layout shift.